The sample demo is not working on MAC OS

**Describe the bug**
The docker-compose up command fails in building the docker images at web-agent.dockerfile.

**To Reproduce**
Steps to reproduce the behavior:
1. clone the repo in a MAC OS
2. go to repo root
3. execute docker-compose up command
4. You get the following error - The referenced project '../../src/AgentFramework.AspNetCore/AgentFramework.AspNetCore.csproj' does not exist. [/src/WebAgent.csproj]

**Expected behavior**
The docker containers are up

Here is an excerpt of the error log

Step 9/15 : RUN dotnet build "WebAgent.csproj" -c Release -o /app
 ---> Running in 4ef39ffe5194
Microsoft (R) Build Engine version 15.9.20+g88f5fadfbe for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  Skipping project "/src/AgentFramework.AspNetCore/AgentFramework.AspNetCore.csproj" because it was not found.
  Skipping project "/src/AgentFramework.AspNetCore/AgentFramework.AspNetCore.csproj" because it was not found.
  Restoring packages for /src/WebAgent.csproj...
  Restoring packages for /src/WebAgent.csproj...
  Restore completed in 384.04 ms for /src/WebAgent.csproj.
  Generating MSBuild file /src/obj/WebAgent.csproj.nuget.g.props.
  Restore completed in 515.86 ms for /src/WebAgent.csproj.
/usr/share/dotnet/sdk/2.2.104/Microsoft.Common.CurrentVersion.targets(1879,5): warning : The referenced project '../../src/AgentFramework.AspNetCore/AgentFramework.AspNetCore.csproj' does not exist. [/src/WebAgent.csproj]
  
  Bundler: Begin processing bundleconfig.json
  Bundler: Done processing bundleconfig.json
Controllers/ConnectionsController.cs(3,14): error CS0234: The type or namespace name 'Reactive' does not exist in the namespace 'System' (are you missing an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(8,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(9,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(10,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(11,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(12,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(13,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(14,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(15,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(16,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(17,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/HomeController.cs(3,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/HomeController.cs(4,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Models/ConnectionsViewModel.cs(2,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Models/CredentialsViewModel.cs(2,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Models/CredentialViewModel.cs(2,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessage.cs(2,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageHandler.cs(3,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageHandler.cs(4,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageHandler.cs(5,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageRecord.cs(2,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessage.cs(2,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(3,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(4,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(6,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(7,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(8,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingResponseMessage.cs(2,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
SimpleWebAgent.cs(2,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Startup.cs(2,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Startup.cs(3,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessage.cs(8,33): error CS0246: The type or namespace name 'AgentMessage' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(199,40): error CS0246: The type or namespace name 'ConnectionInvitationMessage' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(209,16): error CS0246: The type or namespace name 'ConnectionInvitationMessage' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/HomeController.cs(18,13): error CS0246: The type or namespace name 'IWalletService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/HomeController.cs(19,13): error CS0246: The type or namespace name 'IProvisioningService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/HomeController.cs(20,22): error CS0246: The type or namespace name 'WalletOptions' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(39,13): error CS0246: The type or namespace name 'IEventAggregator' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(40,13): error CS0246: The type or namespace name 'IConnectionService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(41,13): error CS0246: The type or namespace name 'IWalletService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(42,13): error CS0246: The type or namespace name 'IWalletRecordService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(43,13): error CS0246: The type or namespace name 'IProvisioningService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(44,13): error CS0246: The type or namespace name 'IAgentProvider' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(45,13): error CS0246: The type or namespace name 'IMessageService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(46,22): error CS0246: The type or namespace name 'WalletOptions' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(29,26): error CS0246: The type or namespace name 'IEventAggregator' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/HomeController.cs(13,26): error CS0246: The type or namespace name 'IWalletService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageHandler.cs(9,40): error CS0246: The type or namespace name 'MessageHandlerBase<>' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageHandler.cs(18,88): error CS0246: The type or namespace name 'IAgentContext' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageHandler.cs(18,116): error CS0246: The type or namespace name 'MessageContext' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageHandler.cs(18,39): error CS0246: The type or namespace name 'AgentMessage' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageHandler.cs(18,53): error CS0115: 'BasicMessageHandler.ProcessAsync(BasicMessage, IAgentContext, MessageContext)': no suitable method found to override [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageHandler.cs(13,36): error CS0246: The type or namespace name 'IWalletRecordService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageHandler.cs(11,26): error CS0246: The type or namespace name 'IWalletRecordService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/HomeController.cs(14,26): error CS0246: The type or namespace name 'IProvisioningService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/HomeController.cs(15,26): error CS0246: The type or namespace name 'WalletOptions' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessage.cs(10,37): error CS0246: The type or namespace name 'AgentMessage' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(12,44): error CS0246: The type or namespace name 'IMessageHandler' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(30,26): error CS0246: The type or namespace name 'IConnectionService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(31,26): error CS0246: The type or namespace name 'IWalletService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(32,26): error CS0246: The type or namespace name 'IWalletRecordService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(36,28): error CS0246: The type or namespace name 'MessageType' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(47,54): error CS0246: The type or namespace name 'IAgentContext' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(47,82): error CS0246: The type or namespace name 'MessageContext' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(47,27): error CS0246: The type or namespace name 'AgentMessage' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(24,40): error CS0246: The type or namespace name 'IEventAggregator' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(24,74): error CS0246: The type or namespace name 'IMessageService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageRecord.cs(11,39): error CS0246: The type or namespace name 'RecordBase' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(17,26): error CS0246: The type or namespace name 'IEventAggregator' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(22,26): error CS0246: The type or namespace name 'IMessageService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingResponseMessage.cs(10,45): error CS0246: The type or namespace name 'AgentMessage' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
SimpleWebAgent.cs(8,35): error CS0246: The type or namespace name 'AgentBase' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
SimpleWebAgent.cs(15,33): error CS0115: 'SimpleWebAgent.ConfigureHandlers()': no suitable method found to override [/src/WebAgent.csproj]
Models/ConnectionsViewModel.cs(10,28): error CS0246: The type or namespace name 'ConnectionRecord' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Models/ConnectionsViewModel.cs(12,28): error CS0246: The type or namespace name 'ConnectionRecord' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Models/ConnectionsViewModel.cs(17,16): error CS0246: The type or namespace name 'ConnectionRecord' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageRecord.cs(13,32): error CS0115: 'BasicMessageRecord.TypeName': no suitable method found to override [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(33,26): error CS0246: The type or namespace name 'IProvisioningService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(34,26): error CS0246: The type or namespace name 'IAgentProvider' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(35,26): error CS0246: The type or namespace name 'IMessageService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(36,26): error CS0246: The type or namespace name 'WalletOptions' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Models/CredentialViewModel.cs(12,16): error CS0246: The type or namespace name 'CredentialState' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]

Build FAILED.

/usr/share/dotnet/sdk/2.2.104/Microsoft.Common.CurrentVersion.targets(1879,5): warning : The referenced project '../../src/AgentFramework.AspNetCore/AgentFramework.AspNetCore.csproj' does not exist. [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(3,14): error CS0234: The type or namespace name 'Reactive' does not exist in the namespace 'System' (are you missing an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(8,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(9,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(10,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(11,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(12,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(13,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(14,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(15,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(16,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(17,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/HomeController.cs(3,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/HomeController.cs(4,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Models/ConnectionsViewModel.cs(2,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Models/CredentialsViewModel.cs(2,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Models/CredentialViewModel.cs(2,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessage.cs(2,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageHandler.cs(3,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageHandler.cs(4,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageHandler.cs(5,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageRecord.cs(2,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessage.cs(2,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(3,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(4,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(6,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(7,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(8,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingResponseMessage.cs(2,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
SimpleWebAgent.cs(2,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Startup.cs(2,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Startup.cs(3,7): error CS0246: The type or namespace name 'AgentFramework' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessage.cs(8,33): error CS0246: The type or namespace name 'AgentMessage' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(199,40): error CS0246: The type or namespace name 'ConnectionInvitationMessage' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(209,16): error CS0246: The type or namespace name 'ConnectionInvitationMessage' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/HomeController.cs(18,13): error CS0246: The type or namespace name 'IWalletService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/HomeController.cs(19,13): error CS0246: The type or namespace name 'IProvisioningService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/HomeController.cs(20,22): error CS0246: The type or namespace name 'WalletOptions' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(39,13): error CS0246: The type or namespace name 'IEventAggregator' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(40,13): error CS0246: The type or namespace name 'IConnectionService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(41,13): error CS0246: The type or namespace name 'IWalletService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(42,13): error CS0246: The type or namespace name 'IWalletRecordService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(43,13): error CS0246: The type or namespace name 'IProvisioningService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(44,13): error CS0246: The type or namespace name 'IAgentProvider' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(45,13): error CS0246: The type or namespace name 'IMessageService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(46,22): error CS0246: The type or namespace name 'WalletOptions' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(29,26): error CS0246: The type or namespace name 'IEventAggregator' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/HomeController.cs(13,26): error CS0246: The type or namespace name 'IWalletService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageHandler.cs(9,40): error CS0246: The type or namespace name 'MessageHandlerBase<>' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageHandler.cs(18,88): error CS0246: The type or namespace name 'IAgentContext' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageHandler.cs(18,116): error CS0246: The type or namespace name 'MessageContext' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageHandler.cs(18,39): error CS0246: The type or namespace name 'AgentMessage' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageHandler.cs(18,53): error CS0115: 'BasicMessageHandler.ProcessAsync(BasicMessage, IAgentContext, MessageContext)': no suitable method found to override [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageHandler.cs(13,36): error CS0246: The type or namespace name 'IWalletRecordService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageHandler.cs(11,26): error CS0246: The type or namespace name 'IWalletRecordService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/HomeController.cs(14,26): error CS0246: The type or namespace name 'IProvisioningService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/HomeController.cs(15,26): error CS0246: The type or namespace name 'WalletOptions' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessage.cs(10,37): error CS0246: The type or namespace name 'AgentMessage' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(12,44): error CS0246: The type or namespace name 'IMessageHandler' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(30,26): error CS0246: The type or namespace name 'IConnectionService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(31,26): error CS0246: The type or namespace name 'IWalletService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(32,26): error CS0246: The type or namespace name 'IWalletRecordService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(36,28): error CS0246: The type or namespace name 'MessageType' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(47,54): error CS0246: The type or namespace name 'IAgentContext' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(47,82): error CS0246: The type or namespace name 'MessageContext' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(47,27): error CS0246: The type or namespace name 'AgentMessage' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(24,40): error CS0246: The type or namespace name 'IEventAggregator' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(24,74): error CS0246: The type or namespace name 'IMessageService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageRecord.cs(11,39): error CS0246: The type or namespace name 'RecordBase' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(17,26): error CS0246: The type or namespace name 'IEventAggregator' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingMessageHandler.cs(22,26): error CS0246: The type or namespace name 'IMessageService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/TrustPing/TrustPingResponseMessage.cs(10,45): error CS0246: The type or namespace name 'AgentMessage' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
SimpleWebAgent.cs(8,35): error CS0246: The type or namespace name 'AgentBase' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
SimpleWebAgent.cs(15,33): error CS0115: 'SimpleWebAgent.ConfigureHandlers()': no suitable method found to override [/src/WebAgent.csproj]
Models/ConnectionsViewModel.cs(10,28): error CS0246: The type or namespace name 'ConnectionRecord' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Models/ConnectionsViewModel.cs(12,28): error CS0246: The type or namespace name 'ConnectionRecord' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Models/ConnectionsViewModel.cs(17,16): error CS0246: The type or namespace name 'ConnectionRecord' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Protocols/BasicMessage/BasicMessageRecord.cs(13,32): error CS0115: 'BasicMessageRecord.TypeName': no suitable method found to override [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(33,26): error CS0246: The type or namespace name 'IProvisioningService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(34,26): error CS0246: The type or namespace name 'IAgentProvider' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(35,26): error CS0246: The type or namespace name 'IMessageService' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Controllers/ConnectionsController.cs(36,26): error CS0246: The type or namespace name 'WalletOptions' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
Models/CredentialViewModel.cs(12,16): error CS0246: The type or namespace name 'CredentialState' could not be found (are you missing a using directive or an assembly reference?) [/src/WebAgent.csproj]
    1 Warning(s)
    82 Error(s)

Time Elapsed 00:00:02.86
ERROR: Service 'web-agent-1' failed to build: The command '/bin/sh -c dotnet build "WebAgent.csproj" -c Release -o /app' returned a non-zero code: 1

**Desktop (please complete the following information):**
 - OS: MACOs Mojave
 - Version 10.14.5

--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -1,0 +1,10 @@
+<Project>
+ <PropertyGroup>
+   <Version>4.0.0</Version>
+   <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <Company>AgentFramework</Company>
+    <Authors>Tomislav Markovski, Tobias Looker</Authors>
+    <PackageProjectUrl>https://github.com/streetcred-id/agent-framework</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/streetcred-id/agent-framework</RepositoryUrl>
+ </PropertyGroup>
+</Project>

--- a/docker-compose.yaml
+++ b/docker-compose.yaml
@@ -19,7 +19,7 @@ services:
     environment:
       - ASPNETCORE_URLS=http://10.0.0.11:7000
       - ASPNETCORE_ENVIRONMENT=Development
-      - ENDPOINT_HOST=$HOST1
+      - ENDPOINT_HOST=${HOST1:-http://10.0.0.11:7000}
     ports:
       - "7000:7000"
     depends_on:
@@ -37,7 +37,7 @@ services:
     environment:
       - ASPNETCORE_URLS=http://10.0.0.12:8000
       - ASPNETCORE_ENVIRONMENT=Development
-      - ENDPOINT_HOST=$HOST2
+      - ENDPOINT_HOST=${HOST2:-http://10.0.0.12:8000}
     ports:
       - "8000:8000"
     depends_on:

--- a/samples/aspnetcore/WebAgent.csproj
+++ b/samples/aspnetcore/WebAgent.csproj
@@ -7,10 +7,6 @@
 
   <PropertyGroup Condition=" '$(RunConfiguration)' == 'Default' ">
     <StartAction>Project</StartAction>
-    <EnvironmentVariables>
-      <Variable name="ASPNETCORE_ENVIRONMENT" value="Development" />
-      <Variable name="ASPNETCORE_ENVIRONMENT" value="Development" />
-    </EnvironmentVariables>
     <ApplicationURL>http://localhost:5000/</ApplicationURL>
   </PropertyGroup>
   <ItemGroup>
@@ -19,6 +15,7 @@
     <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.0.1916590" />
     <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.0.4" />
     <PackageReference Include="Jdenticon.AspNetCore" Version="2.2.1" />
+    <PackageReference Include="AgentFramework.AspNetCore" Version="4.0.0-preview.648" />
   </ItemGroup>
 
   <ItemGroup>
@@ -31,7 +28,4 @@
     </None>
   </ItemGroup>
 
-  <ItemGroup>
-    <ProjectReference Include="..\..\src\AgentFramework.AspNetCore\AgentFramework.AspNetCore.csproj" />
-  </ItemGroup>
 </Project>

--- a/src/AgentFramework.AspNetCore/AgentFramework.AspNetCore.csproj
+++ b/src/AgentFramework.AspNetCore/AgentFramework.AspNetCore.csproj
@@ -2,18 +2,9 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <Version>4.0.0</Version>
-    <AssemblyVersion>4.0.0</AssemblyVersion>
-    <FileVersion>4.0.0</FileVersion>
     <Description>ASP.NET Core support for Agent Framework</Description>
     <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\AgentFramework.AspNetCore.xml</DocumentationFile>
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
-    <PackageLicenseUrl>https://github.com/streetcred-id/agent-framework/blob/master/LICENSE</PackageLicenseUrl>
-    <PackageProjectUrl>https://github.com/streetcred-id/agent-framework</PackageProjectUrl>
-    <RepositoryUrl>https://github.com/streetcred-id/agent-framework</RepositoryUrl>
-    <Company>AgentFramework</Company>
-    <Authors>Tomislav Markovski, Tobias Looker</Authors>
   </PropertyGroup>
 
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
@@ -37,7 +28,4 @@
     <ProjectReference Include="..\AgentFramework.Core\AgentFramework.Core.csproj" />
   </ItemGroup>
 
-  <ItemGroup>
-    <Folder Include="Hosting\" />
-  </ItemGroup>
 </Project>

--- a/src/AgentFramework.Core.Handlers/AgentFramework.Core.Handlers.csproj
+++ b/src/AgentFramework.Core.Handlers/AgentFramework.Core.Handlers.csproj
@@ -2,27 +2,13 @@
 
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
-  </PropertyGroup>
-  <PropertyGroup>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <Version>4.0.0</Version>
-    <AssemblyVersion>4.0.0</AssemblyVersion>
-    <FileVersion>4.0.0</FileVersion>
     <Description>Support for extending AgentFramework using message handlers</Description>
     <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\AgentFramework.Core.Handlers.xml</DocumentationFile>
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
-    <PackageLicenseUrl>https://github.com/streetcred-id/agent-framework/blob/master/LICENSE</PackageLicenseUrl>
-    <PackageProjectUrl>https://github.com/streetcred-id/agent-framework</PackageProjectUrl>
-    <RepositoryUrl>https://github.com/streetcred-id/agent-framework</RepositoryUrl>
-    <Company>AgentFramework</Company>
-    <Authors>Tomislav Markovski, Tobias Looker</Authors>
   </PropertyGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\AgentFramework.Core\AgentFramework.Core.csproj" />
   </ItemGroup>
 
-  <ItemGroup>
-    <Folder Include="Agents\" />
-  </ItemGroup>
 </Project>

--- a/src/AgentFramework.Core/AgentFramework.Core.csproj
+++ b/src/AgentFramework.Core/AgentFramework.Core.csproj
@@ -2,20 +2,9 @@
 
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
-  </PropertyGroup>
-  <PropertyGroup>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <Description>.NET Core tools for building agent services</Description>
-    <Version>4.0.0</Version>
-    <AssemblyVersion>4.0.0</AssemblyVersion>
-    <FileVersion>4.0.0</FileVersion>
     <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\AgentFramework.Core.xml</DocumentationFile>
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
-    <PackageLicenseUrl>https://github.com/streetcred-id/agent-framework/blob/master/LICENSE</PackageLicenseUrl>
-    <PackageProjectUrl>https://github.com/streetcred-id/agent-framework</PackageProjectUrl>
-    <RepositoryUrl>https://github.com/streetcred-id/agent-framework</RepositoryUrl>
-    <Company>AgentFramework</Company>
-    <Authors>Tomislav Markovski, Tobias Looker</Authors>
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Logging" Version="2.1.1" />

--- a/src/AgentFramework.TestHarness/AgentFramework.TestHarness.csproj
+++ b/src/AgentFramework.TestHarness/AgentFramework.TestHarness.csproj
@@ -3,16 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <IsPackable>true</IsPackable>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <Description>A Test Harness for testing AgentFramework</Description>
-    <Version>3.1.0</Version>
-    <AssemblyVersion>3.1.0</AssemblyVersion>
-    <FileVersion>3.1.0</FileVersion>
-    <PackageLicenseUrl>https://github.com/streetcred-id/agent-framework/blob/master/LICENSE</PackageLicenseUrl>
-    <PackageProjectUrl>https://github.com/streetcred-id/agent-framework</PackageProjectUrl>
-    <RepositoryUrl>https://github.com/streetcred-id/agent-framework</RepositoryUrl>
-    <Company>AgentFramework</Company>
-    <Authors>Tomislav Markovski, Tobias Looker</Authors>
   </PropertyGroup>
 
   <ItemGroup>

